def print_error(self,error_str):
    print(f"произошла ошибка: {error_str}")